test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/AuthLdapIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2018 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.integrationtests.ldap;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.kie.cloud.api.scenario.WorkbenchKieServerScenario;
+import org.kie.cloud.api.settings.LdapSettings;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.cloud.integrationtests.category.JBPMOnly;
+import org.kie.cloud.integrationtests.testproviders.FireRulesTestProvider;
+import org.kie.cloud.integrationtests.testproviders.HttpsKieServerTestProvider;
+import org.kie.cloud.integrationtests.testproviders.HttpsWorkbenchTestProvider;
+import org.kie.cloud.integrationtests.testproviders.OptaplannerTestProvider;
+import org.kie.cloud.integrationtests.testproviders.PersistenceTestProvider;
+import org.kie.cloud.integrationtests.testproviders.ProcessTestProvider;
+import org.kie.cloud.integrationtests.util.ScenarioDeployer;
+import org.kie.cloud.maven.constants.MavenConstants;
+
+public class AuthLdapIntegrationTest extends AbstractCloudIntegrationTest {
+
+    private static WorkbenchKieServerScenario deploymentScenario;
+
+    @BeforeClass
+    public static void initializeDeployment() {
+
+        LdapSettings ldapSettings = deploymentScenarioFactory.getLdapSettings()
+                //.withLdapUrl("ldap://bxms-binaries.usersys.redhat.com:389")
+                .withLdapBindDn("cn=Manager,dc=example,dc=com")
```

Comment:
Is it so that there properties ale already hardcoded in running LDAP server without possibility to be changed?
If so they should be put into one place (for example test-cloud-remote pom.xml). What do you think?